src/test/Components/CreateImageWizardV2/CreateImageWizard.compliance.test.tsx

```diff
@@ -154,6 +154,13 @@ describe('Step Compliance', () => {
         /cis red hat enterprise linux 8 benchmark for level 1 - workstation/i
       )
     );
+    screen.getByText(/kernel arguments:/i);
+    screen.getByText(/audit_backlog_limit=8192 audit=1/i);
+    screen.getByText(/disabled services:/i);
+    screen.getByText(/nfs-server/i);
+    screen.getByText(/enabled services:/i);
+    screen.getByText(/enabled services:/i);
```

Comment:
As the test suite is mostly asynchronous let's use the findBy query where possible to avoid flakes due to timing.

```suggestion
    await screen.findByText(/kernel arguments:/i);
    await screen.findByText(/audit_backlog_limit=8192 audit=1/i);
    await screen.findByText(/disabled services:/i);
    await screen.findByText(/nfs-server/i);
    await screen.findByText(/enabled services:/i);
    await screen.findByText(/enabled services:/i);
```